packages/core/src/intl/getFallbackLocale.test.ts

```diff
@@ -0,0 +1,61 @@
+import { Locale } from '@prezly/theme-kit-intl';
+
+import { getFallbackLocale } from './getFallbackLocale';
+
+describe('getFallbackLocale', () => {
+    it('should return selected region-dependent fallback locale for English', () => {
+        expect(
+            getFallbackLocale('en', [
+                Locale.from('en_US'),
+                Locale.from('en_GB'),
+                Locale.from('en_DE'),
+            ]),
+        ).toEqual(Locale.from('en_US'));
+
+        expect(getFallbackLocale('en', [Locale.from('en_GB'), Locale.from('en_DE')])).toEqual(
+            Locale.from('en_GB'),
+        );
+
+        expect(getFallbackLocale('en', [Locale.from('en_DE')])).toBeUndefined();
+    });
+
+    it('should return selected region-dependent fallback locale for French', () => {
+        expect(
+            getFallbackLocale('fr', [
+                Locale.from('en_US'),
+                Locale.from('fr_FR'),
+                Locale.from('fr_CA'),
+                Locale.from('fr_BE'),
+            ]),
+        ).toEqual(Locale.from('fr_FR'));
+
+        expect(
+            getFallbackLocale('fr', [
+                Locale.from('en_US'),
+                Locale.from('fr_CA'),
+                Locale.from('fr_BE'),
+            ]),
+        ).toEqual(Locale.from('fr_CA'));
+
+        expect(getFallbackLocale('fr', [Locale.from('fr_BE')])).toBeUndefined();
```

Comment:
I've checked the code and the only place where this function is used is `getAlternateLanguageLinks()`. 

See the tests for it: [getAlternateLanguageLinks.test.ts](https://github.com/prezly/theme-kit-js/blob/235f1fa84385aeed2c3431b6ad712b053889185b/packages/core/src/seo/getAlternateLanguageLinks.test.ts)

Because the tests still pass, unchanged, as they were initially implemented by Dmitry in #215, I don't think we should change anything here.

Reply:
Additionally, I've moved the `getFallbackLocale()` function closer to `getAlternateLanguageLinks()` and made it private to the package (no public export anymore). Just to make sure we don't use this confusing function externally.

26ca6b9efd8967e0f170e28df5bc5781834daee1